### PR TITLE
Improve sql bbe to include queryRow sample

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2840

### CODE REVIEW

File: examples/jdbc-query-operation/jdbc_query_operation.bal [hunk 1]
```diff
@@ -6,7 +6,7 @@ import ballerina/sql;
 // 'getDataWithTypedQuery' function. In this example, all columns of the 
 // customer table will be loaded. Therefore, the `Customer` record will be 
 // created with all the columns. The column name of the result and the 
-// defined field name of the record will be matched case insensitively.
+// defined field name of the record will be matched regardless of letters' case.
```
Comment (praneesha) [inline examples/jdbc-query-operation/jdbc_query_operation.bal:9]:
```suggestion
// defined field name of the record will be matched regardless of the letters' case.
```

File: examples/jdbc-query-operation/jdbc_query_operation.bal [hunk 2]
```diff
@@ -28,46 +28,31 @@ public function main() returns error? {
...
+        `SELECT * FROM Customers where customerId = ${customerId}`);
+    io:println("\nCustomer (customerId = 1) : ", customer);
 
-    // Iterates the customer stream.
-    check from Customer customer in customerStream
-        do {
-            io:println("Full Customer details: ", customer);
-        };
+    // The result of the count operation is provided as an int variable.
+    // Since this query only returns a single column on top of single row,
```
Comment (praneesha) [inline examples/jdbc-query-operation/jdbc_query_operation.bal:51]:
```suggestion
    // As this query returns only a single column on top of a single row,
```

File: examples/mysql-query-operation/mysql_query_operation.bal [hunk 3]
```diff
@@ -6,7 +6,7 @@ import ballerina/sql;
 // 'getDataWithTypedQuery' function. In this example, all columns of the 
 // customer table will be loaded. Therefore, the `Customer` record will be 
 // created with all the columns. The column name of the result and the 
-// defined field name of the record will be matched case insensitively.
+// defined field name of the record will be matched regardless of letters' case.
```
Comment (praneesha) [inline examples/mysql-query-operation/mysql_query_operation.bal:9]:
```suggestion
// defined field name of the record will be matched regardless of the letters' case.
```

File: examples/mysql-query-operation/mysql_query_operation.bal [hunk 4]
```diff
@@ -21,55 +21,40 @@ public function main() returns error? {
...
-    check from Customer customer in customerStream
+    check from Customer customer in resultStream
         do {
             io:println("Full Customer details: ", customer);
         };
 
+    int customerId = 1;
+    // Select a row in the database table via the query row operation.
+    // This will return utmost one record. If no record is found, it will
+    // throw sql:NoRowsError.
```
Comment (praneesha) [inline examples/mysql-query-operation/mysql_query_operation.bal:46]:
```suggestion
    // throw an `sql:NoRowsError`.
```

**Comment by praneesha** [comment]
@niveathika - As these changes are targeted at the 2201.0.1 release, please check with @keizer619 to which branch these BBE changes should be sent to.

**Review by praneesha — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Comment by praneesha** [comment]
> @praneesha This is targeted for Update 1

Ack.Thanks.

**Pull request merged**